site/smart-accounts/light-account/index.md

```diff
@@ -33,18 +35,46 @@ The code snippet below demonstrates how to use Light Account with Account Kit. I
+
+For backwards compatibility, Light Account defaults to version v1.1.0, but we recommend v2.0.0 for new dapps because of the advantages that come with entry point v0.7. However, once a version is chosen and the light account is created, the version must remain consistent in order for the light account client to work with the existing light account.
+
 ::: tip Address calculation
-For the Light Account, the address of the smart account will be calculated as a combination of [the owner and the salt](https://github.com/alchemyplatform/light-account/blob/main/src/LightAccountFactory.sol#L24-L33). You will get the same smart account address each time you supply the same `owner`, the signer used to create the account for the first time. Alternatively, you can supply `salt` if you want a different address for the same `owner` param (the default salt is `0n`).
+For the Light Account, the address of the smart account will be calculated as a combination of the version, [the owner, and the salt](https://github.com/alchemyplatform/light-account/blob/main/src/LightAccountFactory.sol#L24-L33). You will get the same smart account address each time you supply the same `version` and `owner`. Alternatively, you can supply `salt` if you want a different address for the same `version` and `owner` params (the default salt is `0n`).
 
 If you want to use a signer to connect to an account whose address does not map to the contract-generated address, you can supply the `accountAddress` to connect with the account of interest. In that case, the `signer` address is not used for address calculation, but only used for signing the operation.
 
 Reference: https://eips.ethereum.org/EIPS/eip-4337#first-time-account-creation
 :::
 
-## [Deployment addresses](https://github.com/alchemyplatform/light-account/blob/v1.1.0/Deployments.md)
+## [Deployment addresses](https://github.com/alchemyplatform/light-account/tree/v2.0.0/deployments)
 
 The following table lists the deployed factory and account implementation contract addresses for `LightAccount` on different chains:
 
+### v2.0.0
+
+| Chain            | Factory Address                            | Account Implementation                     |
+| ---------------- | ------------------------------------------ | ------------------------------------------ |
+| Eth Mainnet      | 0x0000000000400CdFef5E2714E63d8040b700BC24 | 0x8E8e658E22B12ada97B402fF0b044D6A325013C7 |
+| Eth Sepolia      | 0x0000000000400CdFef5E2714E63d8040b700BC24 | 0x8E8e658E22B12ada97B402fF0b044D6A325013C7 |
+| Polygon Mainnet  | 0x0000000000400CdFef5E2714E63d8040b700BC24 | 0x8E8e658E22B12ada97B402fF0b044D6A325013C7 |
+| Polygon Mumbai   | 0x0000000000400CdFef5E2714E63d8040b700BC24 | 0x8E8e658E22B12ada97B402fF0b044D6A325013C7 |
```

Comment:
We can take Mumbai out, it's not deployed there and the network is deprecated similar to Goerli.